database/migrations/2019_07_16_093502_config_upgrade.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/** @noinspection PhpUndefinedClassInspection */
+use App\Configs;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+
+class ConfigUpgrade extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		if (Schema::hasTable('configs')) {
+			Configs::where('key', '=', 'checkForUpdates')->update(['key' => 'check_for_updates']);
+			Configs::where('key', '=', 'sortingPhotos_col')->update(['key' => 'sorting_Photos_col']);
+			Configs::where('key', '=', 'sortingPhotos_order')->update(['key' => 'sorting_Photos_order']);
+			Configs::where('key', '=', 'sortingAlbums_col')->update(['key' => 'sorting_Albums_col']);
+			Configs::where('key', '=', 'sortingAlbums_order')->update(['key' => 'sorting_Albums_order']);
+			Configs::where('key', '=', 'skipDuplicates')->update(['key' => 'skip_duplicates']);
+			Configs::where('key', '=', 'deleteImported')->update(['key' => 'delete_imported']);
+
+			define('INT', 'int');
+			define('STRING', 'string');
+			define('STRING_REQ', 'string_required');
+			define('BOOL', '0|1');
+			define('TERNARY', '0|1|2');
+			define('DISABLED', '');
+
+			Schema::table('configs', function (Blueprint $table) {
+				$table->string('type_range')->after('cat')->default('0|1');
+				$table->string('description')->after('confidentiality')->default('');
+			});
+
+			Configs::where('key', '=', 'version')->update(['type_range' => INT]);
+			Configs::where('key', '=', 'username')->update(['type_range' => STRING_REQ]);
+			Configs::where('key', '=', 'password')->update(['type_range' => STRING_REQ]);
+			Configs::where('key', '=', 'check_for_updates')->update(['type_range' => BOOL]);
+			Configs::where('key', '=', 'sorting_Photos_col')->update(['type_range' => 'id|takestamp|title|description|public|star|type']);
+			Configs::where('key', '=', 'sorting_Photos_order')->update(['type_range' => 'ASC|DESC']);
+			Configs::where('key', '=', 'sorting_Albums_col')->update(['type_range' => 'id|title|description|public|max_takestamp|min_takestamp']);
+			Configs::where('key', '=', 'sorting_Albums_order')->update(['type_range' => 'ASC|DESC']);
+			Configs::where('key', '=', 'imagick')->update(['type_range' => BOOL]);
+			Configs::where('key', '=', 'dropboxKey')->update(['type_range' => STRING]);
```

Comment:
If we are standardizing on snake case, shouldn't this be `dropbox_key` then (or maybe `Dropbox_key`)?